datasets/stereoset/stereoset.py

```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Lint as: python3
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+_CITATION = """\
+@article{nadeem2020stereoset,
+  title={StereoSet: Measuring stereotypical bias in pretrained language models},
+  author={Nadeem, Moin and Bethke, Anna and Reddy, Siva},
+  journal={arXiv preprint arXiv:2004.09456},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+StereoSet is a dataset that measures stereotype bias in language models. StereoSet consists of 17,000 sentences that
+measures model preferences across gender, race, religion, and profession.
+"""
+
+_LICENSE = "CC BY-SA 4.0"
+
+
+class StereoSetConfig(datasets.BuilderConfig):
+    """BuilderConfig"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(StereoSetConfig, self).__init__(**kwargs)
+
+
+class StereoSet(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Stereoset(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `stereoset`